cmd/thanos/main_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"path/filepath"
 
 	"github.com/thanos-io/thanos/pkg/block/metadata"
+	"github.com/thanos-io/thanos/pkg/testutil/e2eutil"
```

Comment:
nit: Shall we group the imports? Does it even matter for the tests?

Reply:
we should but we don't have tools for that so it's tedious and I am lazy (: 

@jojohappy if you were asking if still valid: https://github.com/thanos-io/thanos/issues/984